### DISCUSSION TITLE
Realtek bluetooth adapter ID: 0bda:8771

### DISCUSSION THREAD

**Discussion by jacktolmie** [SpiralLinux/SpiralLinux-project#149] [General]
This is a fairly common cheap 5.0 Bluetooth adapter chipset. To make it work you need to download the two following drivers:
https://github.com/Realtek-OpenSource/android_hardware_realtek/raw/rtk1395/bt/rtkbt/Firmware/BT/rtl8761b_config
https://github.com/Realtek-OpenSource/android_hardware_realtek/blob/rtk1395/bt/rtkbt/Firmware/BT/rtl8761b_fw

Rename the files to: rtl8761bu_fw.bin and rtl8761bu_config.bin . Copy them over to /lib/firmware/rtl_bt/ .
Unplug and reinsert the usb dongle and it should work fine.
Hopefully this helps someone. I know it took me a while before I found a  solution. I cannot remember the site I found it on, so I cannot give credit to them. Maybe those files could be added into the distro so this does not have to be done manually. I see a lot of users of other distros talking about this issue.

**Comment by geckolinux** [comment]
Hi there, thanks for the useful report! So it looks like that's a newer model, apparently it will be supported in the next Debian stable release:
https://packages.debian.org/bookworm/firmware-realtek (search for `RTL8761BU`) but it's not supported in the current Debian stable version:
https://packages.debian.org/bullseye/firmware-realtek
Until then, another easier way to install support for it would be to simply install the Bookworm version:
https://packages.debian.org/bullseye/all/firmware-realtek/download

**Reply by jacktolmie** [reply]
Thanks. That is good to know. Thank you for looking into that. I will await the next release as I used bookworm before. I prefer stable for not having to reboot as often because of updates :).

**Reply by geckolinux** [reply]
>  I prefer stable for not having to reboot as often because of updates :).

Hi there, yes, I didn't mean upgrade the system to Bookworm, you can just cherry-pick the `firmware-realtek` DEB package from Bookworm and install it with Gdebi.

**Reply by jacktolmie** [reply]
Ah, makes sense. Thank you. I can do that.

**Comment by Senshi111** [comment]
you can find those and all other firmware here, you can just copy to /lib/firmware
https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/tree/rtl_bt

```
mkdir firmware
cd firmware
wget -r -nd -e robots=no -A '*.bin' --accept-regex '/plain/' https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/tree/rtl_bt/
sudo mv *.bin /lib/firmware/rtl_bt/
sudo update-initramfs -c -k all
```

**Reply by jacktolmie** [reply]
Well nice find. Thanks. I will have to do that next time I install. I really appreciate your help.
